dhis-2/dhis-services/dhis-service-tracker/src/main/java/org/hisp/dhis/tracker/deduplication/PotentialDuplicate.java

```diff
@@ -60,32 +58,28 @@ public class PotentialDuplicate extends BaseIdentifiableObject {
 
   public PotentialDuplicate() {}
 
-  public PotentialDuplicate(String original, String duplicate) {
+  public PotentialDuplicate(UID original, UID duplicate) {
     this.original = original;
     this.duplicate = duplicate;
   }
 
   @JsonProperty
   @JacksonXmlProperty
-  @Property(value = PropertyType.IDENTIFIER, required = Property.Value.TRUE)
```

Comment:
Is this information automatically provided by the type being UID? As I think this was exposed in the schema

Reply:
But we are not using the schema here for anything, I am not sure to be honest, but I think that was not required. Anyway, the required property was not working.
I am asking @dhis2/platform-backend for a review on this PR, to especially check on this line and the `UIDUserType`

Reply:
My understanding of how the schemas work is that the line removed here would only ever have worked if there was a schema for `PotentialDuplicate`. And it looks like that does not exist. So this seems fine to me. As long as you're seeing expected behaviour. If there's a test covering that situation where no `PotentialDuplicate` can be missing this property, even better.